examples/payload_v2/eq_runner_to_downstream/surveyresponse_0_0_1.json

```diff
@@ -0,0 +1,37 @@
+{
+    "tx_id": "ea82c224-0f80-41cc-b877-8a7804b56c26",
+    "type": "uk.gov.ons.edc.eq:surveyresponse",
+    "version": "0.0.1",
```

Comment:
Should the version be v2 and does there need to be a data_version?

Reply:
Good spot. Have fixed the example, so version is `v2` and added the `data_version` property also.